lib/ansible/modules/files/acl.py

```diff
@@ -27,12 +27,13 @@
 description:
     - Sets and retrieves file ACL information.
 options:
-  name:
+  path:
     required: true
     default: null
     description:
       - The full path of the file or object.
```

Comment:
Ah, you want that as part of the description ? I added it as a note.

The reason I did not want to put that into the description itself, is that in the documentation you can see the aliases straight away. So it is already obvious both work fine.